README.md

```diff
@@ -53,46 +58,50 @@ Host *.stanford.edu
     GSSAPIDelegateCredentials yes
 ```
 
-### Problems with Authentication?
+### Set Dor-Services-App and Dor-Services Tokens
```

Comment:
There is only one dor-services-app. dor-services is a gem used by dor-services-app.

Reply:
This isn't about the names of apps, it's about the labels in the yaml file.